.github/workflows/release.yaml

```diff
@@ -322,6 +322,37 @@ jobs:
           IMAGE=$(cat release/IMAGE)
           docker push "$IMAGE"
           cosign sign $(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE")
+
+      # https://github.com/aws-actions/configure-aws-credentials?tab=readme-ov-file#assumerole-with-static-iam-credentials-in-repository-secrets
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+          #role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          #role-external-id: ${{ secrets.AWS_ROLE_EXTERNAL_ID }}
+          role-duration-seconds: 1200
+          role-session-name: AWSCIPush
+      - name: Build and push AWS image
+        if: matrix.flavor == 'ubuntu' && matrix.flavorRelease == '24.04' && matrix.variant == 'standard' && matrix.model == 'generic' && matrix.arch == 'amd64'
```

Comment:
This will match all 3 k3s versions. I don't see any way to upload only one without hardcoding it (or doing the things I deleted in this PR).

The other option is to upload the code image but if the user tries to "reset" to a standard one using userdata (which is a very possible thing), the partitions won't be big enough for the standard image. They can specify sizes for the partitions I think but the recovery one can't be resized cause it's what is booted that point.

Any ideas?

Reply:
We discussed this in Slack and decided to just push a core image. We will then implement a flag in auroraboot to set the "minimum image size" which will override the calculated one when building the cloud image. This way we can set this to a hardcoded value that can fit all our image (standard or core). TODO: Add ticket link